### PR TITLE
UHF-1772 Missing integration overviews

### PR DESCRIPTION
**Pull request by tapsal**
Testing:
* If you don't already have a site for testing, create one:
  * `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
  * cd hel-platform
  * make new
* Update the module to include changes: `composer require drupal/helfi_tpr:dev-UHF-1772-missing-integration-overviews`
* Run `drush updb`
* Make sure you have imported some "Errand service" and "Service channel" content. If not, run:
  * `drush migrate:import tpr_errand_service`
  * `drush migrate:import tpr_service_channel`
* As an anonymous user, go to the pages `/admin/content/integrations/tpr-errand-service` and `admin/content/integrations/tpr-service-channel`. You should **not** see any content.
* As an admin user (for testing purposes, use a user whose UID is **not** 1), go to the same pages and you **should see** the list of the migrated content.
  * Test that it's possible to publish multiple items using the bulk update functionality.

Also check the site specific PRs:
* [drupal-helfi/pull/168](https://github.com/City-of-Helsinki/drupal-helfi/pull/168)
* [drupal-helfi-sote/pull/55](https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/55)
* [drupal-helfi-kymp/pull/51](https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/51)

### CODE REVIEW

File: tests/src/Functional/ErrandServiceListTest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * Tests collection route (views).
+   */
+  public function testList() : void {
```
Comment (tuutti) [inline tests/src/Functional/ErrandServiceListTest.php:35]: This whole thing could probably be refactored into a generic `ListTestBase` class that has `testList()` method. Then make all `*ListTest`s to extend that class with some way to pass admin url and required permissions to that method.

File: tests/src/Functional/ServiceChannelListTest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * Tests collection route (views).
+   */
+  public function testList() : void {
```
Comment (tuutti) [inline tests/src/Functional/ServiceChannelListTest.php:35]: Same as above.

File: tests/src/Functional/ServiceChannelListTest.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    $this->listPermissions = [
+      'access remote entities overview',
+    ];
+    $this->adminListPath = '/admin/content/integrations/tpr-service-channel';
+  }
+
+  /**
+   * Tests list view permissions.
+   */
+  public function testListPermissions() : void {
```
Comment (tuutti) [inline tests/src/Functional/ServiceChannelListTest.php:28]: This can be removed if you change the parent testList() to be public (I think).

**Review by ltpk — APPROVED** [review]
To the moon!

**Pull request merged**